cs/src/Contracts/TunnelHeaderNames.cs

```diff
@@ -27,4 +27,9 @@ public static class TunnelHeaderNames
     /// Github Ssh public key which can be used to validate if it belongs to tunnel's owner.
     /// </summary>
     public const string XGithubSshKey = "X-Github-Ssh-Key";
+
+    /// <summary>
+    /// Header that will skip the antiphishing page when connection to a tunnel through web forwarding.
+    /// </summary>
+    public const string XTunnelSkipPhishing = "X-Tunnel-Skip-Phishing-Page";
```

Comment:
The name sounds a little like it's skipping a malicious page. Maybe it should be `X-Tunnel-Skip-AntiPhishing-Page` or `X-Tunnel-Skip-Interstitial -Page`?

Reply:
updated